sql/table.cc

```diff
@@ -3658,10 +3656,10 @@ void free_blob_buffers_and_reset(TABLE *table, uint32 size)
   @retval The form position.
 */
 
-static ulong get_form_pos(File file, uchar *head)
+ulong get_form_pos(File file, uchar *head)
 {
   uchar *pos, *buf;
-  uint names, length;
+  uint   names, length;
```

Comment:
Spurious change

Reply:
Yes, this is autoformatting with clang-format-diff. Removed as discussed.